low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1

```diff
@@ -101,6 +101,7 @@ IP_addr=172.16.1.222
 max_login_attempts=2
 
 [RAIDSENSOR]
+monitor=true # RAIDSENSOR
```

Comment:
why do we need these comments (like # RAIDSENSOR here) all over in config, does code depend on it. Had raised same earlier

Reply:
The value of the 'monitor' key is changed using a shell script, specifically using 'sed'.
That works most naturally on single lines.

The comments help to enable/disable the value for the right sections. It should be possible to pick up the section name from the previous line, but that would be a bit more involved.

Reply:
>The comments help to enable/disable the value for the right sections. It should be possible to pick up the section name from the previous lines, but that would make the sed command somewhat more complicated.

thats needed, or if someone changes the comment, code will break, which is not right. So pick sections first & then run sed for respective monitor line.